docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -0,0 +1,441 @@
+        )
+        # create a byol model based on ResNet
+        self.resnet_byol = \
+            lightly.models.BYOL(self.backbone, num_ftrs=num_ftrs)
+        self.criterion = lightly.loss.SymNegCosineSimilarityLoss()
+
+        self.nn_replacer = NNMemoryBankModule(size=nn_size)
+
+    def forward(self, x):
+        self.resnet_byol(x)
+
+    def training_step(self, batch, batch_idx):
+        (x0, x1), _, _ = batch
+        (z0, p0), (z1, p1) = self.resnet_byol(x0, x1)
+        z0 = self.nn_replacer(z0.detach(), update=False)
+        z1 = self.nn_replacer(z1.detach(), update=True)
+        loss = self.criterion((z0, p0), (z1, p1))
+        self.log('train_loss_ssl', loss)
+        return loss
+
+    def configure_optimizers(self):
+        optim = torch.optim.SGD(self.resnet_byol.parameters(), lr=6e-2,
+                                momentum=0.9, weight_decay=5e-4)
+        scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optim, max_epochs)
+        return [optim], [scheduler]
+
+model_names = ['MoCo_256', 'SimCLR_256', 'SimSiam_256', 'BarlowTwins_256', 
+               'BYOL_256', 'NNCLR_256', 'NNSimSiam_256', 'NNBYOL_256']
+
+models = [MocoModel, SimCLRModel, SimSiamModel, BarlowTwinsModel, 
+          BYOLModel, NNCLRModel, NNSimSiamModel, NNBYOLModel]
+
+bench_results = []
+gpu_memory_usage = []
+
+# loop through configurations and train models
+for batch_size in batch_sizes:
+    for model_name, BenchmarkModel in zip(model_names, models):
+        runs = []
+        for seed in range(n_runs):
+            pl.seed_everything(seed)
+            dataloader_train_ssl, dataloader_train_kNN, dataloader_test = get_data_loaders(batch_size)
+            benchmark_model = BenchmarkModel(dataloader_train_kNN, classes)
+
+            logger = TensorBoardLogger('imagenette_runs', version=model_name)
+
+            trainer = pl.Trainer(max_epochs=max_epochs, 
+                                gpus=gpus,
+                                logger=logger,
+                                distributed_backend=distributed_backend,
+                                default_root_dir=logs_root_dir)
+            trainer.fit(
+                benchmark_model,
+                train_dataloader=dataloader_train_ssl,
+                val_dataloaders=dataloader_test
+            )
+            gpu_memory_usage.append(torch.cuda.max_memory_allocated())
+            torch.cuda.reset_peak_memory_stats()
+            runs.append(benchmark_model.max_accuracy)
+
+            # delete model and trainer + free up cuda memory
+            del benchmark_model
+            del trainer
+            torch.cuda.empty_cache()
+        bench_results.append(runs)
+
+for result, model, gpu_usage in zip(bench_results, model_names, gpu_memory_usage):
+    result_np = np.array(result)
+    mean = result_np.mean()
+    std = result_np.std()
+    print(f'{model}: {mean:.2f} +- {std:.2f}, GPU used: {gpu_usage / (1024.0**3):.1f} GByte', flush=True)
```

Comment:
I would put the term 1024.0**3 in a separate variable, e.g. `BYTES_PER_GIGABYTE=1024.0**3`